package.json

```diff
@@ -63,6 +63,7 @@
     "cheerio": "^1.0.0-rc.12",
     "edn-data": "^1.0.0",
     "find-up": "^6.3.0",
+    "git-url-parse": "^13.1.1",
```

Comment:
Let's avoid this dependency altogether and remove the console.log

Reply:
which console.log are we talking about?